example/app.json

```diff
@@ -28,6 +28,14 @@
       "favicon": "./assets/favicon.png"
     },
     "plugins": [
+      [
+        "expo-camera",
+        {
+          "cameraPermission": "Allow RNGH example to access your camera",
+          "microphonePermission": "Allow RNGH example to access your microphone",
+          "recordAudioAndroid": true
```

Comment:
Which example needs audio access? 😅

Reply:
I think it's for the camera

Reply:
As far as I can see, camera works without those two 😅

Reply:
I probably copied it from the docs: https://docs.expo.dev/versions/latest/sdk/camera/#example-appjson-with-config-plugin.

I assume the app doesn't ask for microphone permissions?

Reply:
Yup, it doesn't. Also I've removed those permissions from `app.json` and it works fine (though I have not looked into other examples, I just assumed that they don't need that 😅).